MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -119,7 +119,11 @@ class RouteManeuverViewController: UIViewController {
     
     public func updateStreetNameForStep() {
         if let name = step?.names?.first {
-            streetLabel.unabridgedText = name
+            if let step = step, let ref = step.codes?.first, ["i", "us"].contains(ref.lowercased().components(separatedBy: " ").first!) {
```

Comment:
<img width="346" alt="screen shot 2017-06-20 at 2 29 34 pm" src="https://user-images.githubusercontent.com/664805/27357135-e09e9078-55c5-11e7-9c0b-73d9a12a7a5c.png">
Could we filter and display the US-101 for segments of type `highway` and use the ceremonial name in other cases? `Redwood Highway` in this screenshot is confusing, but to your point the other street names should show their proper names.